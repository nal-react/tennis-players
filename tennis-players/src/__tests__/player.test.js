import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Player from '../components/player';

// it('renders welcome message', () => {
//   const { getByText } = render(<Player />);
//   expect(getByText('Welcome to React')).toBeInTheDocument();
let player = 
      {
        "firstname": "Stan",
        "lastname": "Wawrinka",
        "shortname": "S.WAW",
        "sex": "M",
        "country": {
          "picture": "https://i.eurosport.com/_iss_/geo/country/flag/large/2213.png",
          "code": "SUI"
        },
        "picture": "https://i.eurosport.com/_iss_/person/pp_clubteam/large/325225.jpg",
        "data": {
          "rank": 21,
          "points": 1784,
          "weight": 81000,
          "height": 183,
          "age": 33,
          "last": [
            1,
            1,
            1,
            0,
            1
          ]
        }
      };

  it('renders correctly', () => {
    const tree = renderer
      .create(<Player {...player}> </Player>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });