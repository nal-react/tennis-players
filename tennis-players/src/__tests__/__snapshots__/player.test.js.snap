// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="player"
>
  <div
    className="player-identity"
  >
    <img
      className="picture"
      src="https://i.eurosport.com/_iss_/person/pp_clubteam/large/325225.jpg"
    />
    <img
      alt="SUI"
      className="flag"
      src="https://i.eurosport.com/_iss_/geo/country/flag/large/2213.png"
    />
    <div>
      <span>
        Stan Wawrinka
      </span>
    </div>
  </div>
  <div
    className="player-stat"
  >
    <div>
      <span
        className="player-stat-name"
      >
        Rank
      </span>
      <p
        className="player-stat-value"
      >
        21
      </p>
    </div>
    <div>
      <span
        className="player-stat-name"
      >
        Points
      </span>
      <p
        className="player-stat-value"
      >
        1784
      </p>
    </div>
    <div>
      <span
        className="player-stat-name"
      >
        Weight
      </span>
      <p
        className="player-stat-value"
      >
        810 Kg
      </p>
    </div>
    <div>
      <span
        className="player-stat-name"
      >
        Age
      </span>
      <p
        className="player-stat-value"
      >
        33
      </p>
    </div>
    <div>
      <span
        className="player-stat-name"
      >
        Last
      </span>
      <p
        className="player-stat-value"
      >
        1
        1
        1
        0
        1
      </p>
    </div>
  </div>
</div>
`;
